examples/datasets/plot_spm_faces_dataset.py

```diff
@@ -5,13 +5,14 @@
 ==========================================
```

Comment:
the dSPM I get running this example do not get the graphs on the website:

http://martinos.org/mne/dev/auto_examples/datasets/plot_spm_faces_dataset.html#sphx-glr-auto-examples-datasets-plot-spm-faces-dataset-py

It's even quite different.